components/humantask/org.wso2.carbon.humantask/src/main/java/org/wso2/carbon/humantask/core/HumanTaskServer.java

```diff
@@ -163,10 +170,10 @@ private void initDataSource() throws HumanTaskServerException {
         //TODO - need to handle the external transaction managers.
         database.setTransactionManager(tnxManager);
         try {
-	        database.start();
+            database.start();
         } catch (Exception e) {
             String errMsg = "Humantask Database Initialization failed.";
-            log.error(errMsg);
+            log.error(errMsg, e);
```

Comment:
Removed additional log.Updated in commit https://github.com/himasha/carbon-business-process/commit/5d5e8c706473cc59f1e1dc0c7360ca5f9b7c3832